sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/state/AsynchronousMetricStorage.java

```diff
@@ -124,7 +145,11 @@ private AsynchronousMetricStorage(
   }
 
   @Override
-  public MetricData collectAndReset(long startEpochNanos, long epochNanos) {
+  public MetricData collectAndReset(
+      CollectionHandle collector,
+      Set<CollectionHandle> allCollectors,
```

Comment:
it's a little weird, from an API perspective that these need to be passed in, but aren't ever used.

Reply:
aren't used.... yet.


I need to solve async instrument DELTA temporality.  Java (today) has some bugs around DELTA metrics, but it does *try* to do delta sums, where you need to remember last-reported value per-collector.  `allCollectors` was there for dynamic registration and cleanup, but can be removed now that we aren't doing that.